lib/gitlab/ci/templates/pipeline/GradleInstall4JPipeline.yml

```diff
@@ -5,14 +5,14 @@ variables:
   DEV_OR_RELEASE_REGEX: '^develop$|^[0-9]+\.[0-9]+$|^release\/.+$'
```

Comment:
If this merge works, please also merge into the new release/1.3 branch since that is our latest main branch.